crates/octez/src/endpoint.rs

```diff
@@ -18,6 +18,12 @@ impl Endpoint {
     }
 }
 
+impl Default for Endpoint {
+    fn default() -> Self {
+        Self::localhost(80)
```

Comment:
q: is 80 the default port for web server? was wondering if this is relevant to octez binaries

Reply:
This has nothing to do with octez binaries. 80 is the default port for http.